### PR TITLE
Update configuration.markdown

### PR DESCRIPTION
**Pull request by spotlesstofu**
Add path to configuration file on the SD card.

## Proposed change

It's useful to know where to find the configuration.yaml file on the SD card.
I was able to rescue my system after discovering this on a forum.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration.markdown [hunk 1]
```diff
@@ -26,6 +26,8 @@ _If you use Home Assistant Operating System, you can find `configuration.yaml` i
 
 _If you use Home Assistant Core, you can find `configuration.yaml` in the config folder passed to the `hass` command (default is `~/.homeassistant`)._
 
+_If you mount the Home Assistant Operating System SD Card on your computer, you can find `/supervisor/homeassistant/configuration.yaml` in the `hassos-data` partition._
```
Comment (frenck) [inline source/_docs/configuration.markdown:29]: I don't think that this is in context here. This isn't really a normal route to edit config files?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/configuration.markdown (hunk 1 above)
Comment (spotlesstofu) [reply source/_docs/configuration.markdown:29]: This is not a common route to edit config files. It's useful to rescue your system when other routes fail. The three preceding lines are about advanced routes too. That's why I believe this is where people would look for such an information (as I did).

Comment (frenck) [reply source/_docs/configuration.markdown:29]: > It's useful to rescue your system when other routes fail. 

It should not be able to get in such situation, and thus you are documenting a workaround to a bug. Did you report this issue in which you can get into this situation?

**Comment by frenck** [comment]
Because there hasn't been any activity on this PR for quite some time now, I've decided to close it for being stale.

Feel free to re-open this PR when you are ready to pick up work on it again 👍

**Pull request closed without merging**